static/js/lib/urls.ts

```diff
@@ -22,3 +22,10 @@ export const siteApiCollaboratorsDetailUrl = (
   name: string,
   username: string
 ): string => `${siteApiCollaboratorsUrl(name)}${username}/`
+export const siteAddContentUrl = (name: string, configName: string): string =>
+  `/sites/${name}/${configName}/add/`
```

Comment:
is maybe `contentType` possibly a more accurate name than `configName` here?

Reply:
just wondering based on how I'm seeing it used

Reply:
Yes that makes sense to me